packages/opentelemetry-sdk-setup/src/sdk-setup.ts

```diff
@@ -0,0 +1,104 @@
+import { HoneycombSDK } from '@honeycombio/opentelemetry-node'
+import { trace, diag, context, propagation, DiagLogLevel, TraceFlags } from '@opentelemetry/api'
+import { Resource } from '@opentelemetry/resources'
+import { SemanticResourceAttributes } from '@opentelemetry/semantic-conventions'
+import type { PackageJson } from 'read-pkg-up'
+
+import { getOtelLogger, loadBaggageFromFile } from './util.js'
+
+export type TracingOptions = {
+  /** This is a temporary property to signal preloading is enabled, can be replaced with `enabled` once we retire build's internal sdk setup */
+  preloadingEnabled: boolean
```

Comment:
This should allow us to rollout this new module without conflicting with the ongoing `tracing` setup in `@netlify/build`